src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/dj/DefaultJudgmentOrderFormGenerator.java

```diff
@@ -77,7 +77,7 @@ private DefaultJudgmentSDOOrderForm getDefaultJudgmentFormHearing(CaseData caseD
             .disposalHearingBundleDJ(caseData.getDisposalHearingBundleDJ())
             .disposalHearingBundleDJAddSection(nonNull(caseData.getDisposalHearingBundleDJ()))
             .typeBundleInfo(nonNull(caseData.getDisposalHearingBundleDJ())
-                                ? fillTypeBundleInfo(caseData.getDisposalHearingBundleDJ().getType()) : null)
+               ? fillTypeBundleInfo(caseData.getDisposalHearingBundleDJ().getType()) : null)
```

Comment:
I wouldn't expect to see a change to this file

Reply:
Should be fixed now.